packages/lesswrong/components/Layout.tsx

```diff
@@ -23,6 +23,7 @@ import type { ToCData, ToCSection } from '../server/tableOfContents';
 import { ForumOptions, forumSelect } from '../lib/forumTypeUtils';
 import { userCanDo } from '../lib/vulcan-users/permissions';
 import { getUserEmail } from "../lib/collections/users/helpers";
+import NoSSR from 'react-no-ssr';
```

Comment:
Our codebase seems divided between material-ui-core and react-no-ssr for importing NoSSSR